test/e2e/serial/tests/tolerations.go

```diff
@@ -841,11 +841,18 @@ func sriovToleration() corev1.Toleration {
 }
 
 func waitForMcpUpdate(cli client.Client, ctx context.Context, mcpsInfo []mcpInfo, updateType MCPUpdateType) {
```

Comment:
this is a good idea but I'm puzzled by the implementation. We don't have `GinkgoHelper()` calls anymore, so failures should point to the specific test, should not they?

Reply:
without the GinkgoHelper() call in a "helper" function, one will be able to see the exact failing line in the nested function and not the line of where this function is being called. 
IOW if inside a spec there is a call for a helper function in line 5, and that helper function has GinkgoHelper(); if the function fails on one of its assertions say line 91 (let's assume it's in the same file), the report will print out that the spec has failed in line 5. Whereas if GinkgoHelper() was to called, the report will print out that the spec has failed in line 91.